includes/class-wc-stripe-webhook-handler.php

```
@@ -1266,13 +1266,24 @@ public function process_payment_intent( $notification ) {
 		$order  = $this->get_order_from_intent( $intent );
 
-		$checkout_type = $intent->metadata->checkout_type ?? '';
+		$checkout_type                       = $intent->metadata->checkout_type ?? '';
+		$resolved_order_via_checkout_session = false;
+		$checkout_session_intent_event_types = [
+			'payment_intent.payment_failed',
+			'payment_intent.succeeded',
+			'payment_intent.amount_capturable_updated',
+		];
+		$is_adaptive_pricing_checkout_intent = WC_Stripe_Checkout_Sessions_Ajax_Handler::ADAPTIVE_PRICING_CHECKOUT_TYPE === $checkout_type;
+		$should_resolve_via_checkout_session = ! $order
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $should_resolve_via_checkout_session. Keep variable name length under 20. (undefined)

(LongVariable)